src/test/kotlin/com/jetbrains/snakecharm/SnakemakeResolveTest.kt

```diff
@@ -8,20 +8,6 @@ import com.jetbrains.python.psi.PyFunction
 import com.jetbrains.snakecharm.fixtures.SnakemakeResolveTestCase
 
```

Comment:
Please remove "unused imports" that appeared due to this cleanup. It is good practice "do not commit yellow code", so it is recommended to resolve code insight warning before commit
![image](https://user-images.githubusercontent.com/72933/56853436-9f17e200-6930-11e9-94bf-9b6c34eb052e.png)